### PR TITLE
PLAT-134020: Use sandstone as default template

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Make Sandstone theme as default for enact create after Sandstone open

### Resolution
Switches back to sandstone template for default.

### Links
PLAT-134020

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
@hong6316 Please write a change log.

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "@babel/preset-typescript": "7.12.1",
     "@babel/runtime": "7.12.1",
     "@enact/dev-utils": "4.0.0",
-    "@enact/template-moonstone": "4.0.0-alpha.1",
+    "@enact/template-sandstone": "2.0.0-alpha.1",
```
Comment (MikyungKim) [inline package.json:59]:
```suggestion
    "@enact/template-sandstone": "1.0.0",
```

Comment (hong6316) [reply package.json]: OK

**Review by MikyungKim — COMMENTED** [review]
Once @enact/template-sandstone released, package-lock.json needs to be updated.

**Comment by hong6316** [comment]
> Once @enact/template-sandstone released, package-lock.json needs to be updated.

agree

**Review by MikyungKim — APPROVED** [review]
:shipit:

**Pull request merged**